nav2_costmap_2d/src/costmap_layer.cpp

```diff
@@ -86,6 +86,11 @@ void CostmapLayer::updateWithTrueOverwrite(
   if (!enabled_) {
     return;
   }
+
+  if (costmap_ == nullptr) {
+    throw "Can't update costmap layer. It has't been initilized yet. Check if /map topic is published.";
```

Comment:
```suggestion
    throw "Can't update costmap layer. It has't been initilized yet.";
```

This class has no idea why the costmap isn't set. This error could be generated when somebody writes a custom costmap plugin. The specific error, helpful error message should be in static_layer where the exception is caught.

Reply:
fixed in latest commits